frontend/src/RecordingPageComp/SimilarSounds.jsx

```diff
@@ -6,8 +6,14 @@ import "./SimilarSounds.css";
 function SimilarSounds({ playbackDuration }) {
   const duration = Math.floor(playbackDuration);
   const [soundList, setSoundList] = useState([]);
+  const [isLoadingSounds, setIsLoadingSounds] = useState(false);
 
   const fetchSounds = async () => {
+    setIsLoadingSounds(true); // Start loading
+    // Simulate loading time or wait for an actual event that confirms the instrument is ready
+    setTimeout(() => {
+      setIsLoadingSounds(false); // Stop loading after 1 second
+    }, 2000);
```

Comment:
You probably don't need to set a timeout here and can just rely on the API response to reset your loading state. See this example here: https://rapidapi.com/guides/loading-state-react